linkerd/app/src/identity.rs

```diff
@@ -30,8 +41,14 @@ pub struct TlsParams {
     pub trust_anchors_pem: String,
 }
 
+#[derive(Clone)]
+pub enum Addr {
+    Spire(Arc<String>),
+    Linkerd(control::ControlAddr),
+}
```

Comment:
Is it possible to move this into the Provider type to avoid having redundant enums?

Reply:
This is used in the `App`, [here](https://github.com/linkerd/linkerd2-proxy/pull/2580/files#diff-4689f302ab1ea17c575cc6e99ad1906a82f3fa3d387b945fe3c25567a6f5635aR351). It was returning a  `control::ControlAddr` before.